test/functional/services/flyout.ts

```diff
@@ -31,20 +31,17 @@ export class FlyoutService extends FtrService {
   }
 
   public async ensureAllClosed(): Promise<void> {
-    const flyoutElements = await this.find.allByCssSelector('.euiFlyout');
```

Comment:
`find.allByCssSelector` uses `timeout.find` (10 sec) by default and if there are no  `.euiFlyout` in DOM, it waits those 10 sec to complete search.